### PR TITLE
Use unstable API version for checksums support

### PR DESCRIPTION
**Pull request by andyw8**
We expect checksums to be part of the 2020-10 release. Until then, we'll need to point to the unstable version of the API.

### Warn Checklist
- [ ] This changes the interface and requires a Major/Minor version change.
- [x] I have :tophat:'d these changes by using the commands I changed by hand.
- [ ] I have added a dependancy to the project.

### CODE REVIEW

File: src/shopify/theme_client.go [hunk 1]
```diff
@@ -344,7 +344,7 @@ func (c Client) DeleteAsset(asset Asset) error {
 func (c Client) assetPath(query map[string]string) string {
 	formatted := "/admin/assets.json"
```
Comment (andyw8) [inline src/shopify/theme_client.go:345]: Missed this and and a couple more, updating...

**Comment by andyw8** [comment]
@tanema 👍 done in c072b67

File: src/shopify/theme_client_test.go [hunk 2]
```diff
@@ -16,6 +16,8 @@ import (
 
 var NoHeaders = map[string]string(nil)
 
+// APIPath is the version of the Admin REST API to use
```
Comment (tanema) [inline src/shopify/theme_client_test.go:19]: Does this comment need to be here?

Comment (andyw8) [reply src/shopify/theme_client_test.go:19]: Yeah, the linter complains otherwise 😞

**Pull request merged**